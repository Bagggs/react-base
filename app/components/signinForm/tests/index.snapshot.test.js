import React from 'react';
import renderer from 'react-test-renderer';
import SigninForm from '../';

describe('SigninForm', () => {
  it('renders correctly', () => {
    const signinFormComponent = renderer.create(<SigninForm />).toJSON();

    expect(signinFormComponent).toMatchSnapshot();
  });

  context('when email and password are invalid', () => {
    it('renders form with validation errors', () => {
      // const signinFormComponent = renderer.create(<SigninForm />).toJSON();
      //
      // expect(signinFormComponent).toMatchSnapshot();
    });
  });
});
