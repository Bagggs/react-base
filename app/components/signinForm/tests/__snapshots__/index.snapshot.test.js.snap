// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigninForm renders correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="modal-body"
  >
    <div
      className="form-group"
    >
      <label
        className="control-label"
        htmlFor="email"
      >
        Email
      </label>
      <input
        className="form-control"
        id="email"
        name="email"
        onChange={[Function]}
        type="text"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className="control-label"
        htmlFor="password"
      >
        Password
      </label>
      <input
        autoComplete="off"
        className="form-control"
        id="password"
        name="password"
        onChange={[Function]}
        type="password"
      />
    </div>
    <div
      className="modal-footer"
    >
      <button
        className="btn btn-primary"
        disabled={false}
        type="submit"
      >
        Submit
      </button>
    </div>
  </div>
</form>
`;

exports[`SigninForm when email and password are invalid renders form with validation errors 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="modal-body"
  >
    <div
      className="form-group"
    >
      <label
        className="control-label"
        htmlFor="email"
      >
        Email
      </label>
      <input
        className="form-control"
        id="email"
        name="email"
        onChange={[Function]}
        type="text"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className="control-label"
        htmlFor="password"
      >
        Password
      </label>
      <input
        autoComplete="off"
        className="form-control"
        id="password"
        name="password"
        onChange={[Function]}
        type="password"
      />
    </div>
    <div
      className="modal-footer"
    >
      <button
        className="btn btn-primary"
        disabled={false}
        type="submit"
      >
        Submit
      </button>
    </div>
  </div>
</form>
`;
